### PR TITLE
Fix sporadic test failure

### PR DESCRIPTION
**Pull request by karjonas**
Sometimes the simulation data is not yet uploaded causing sporadic failures.

### CODE REVIEW

**Comment by karjonas** [comment]
retest this please

File: plugins/CircuitViewer/tests/braynsTestData.cpp [hunk 1]
```diff
@@ -29,12 +29,35 @@
...
 #define BOOST_TEST_MODULE braynsTestData
 #include <boost/test/unit_test.hpp>
 
+#include <chrono>
+#include <thread>
+
+namespace
+{
+constexpr size_t WAIT_MILLISECONDS = 5000;
+const auto waitForSimulationUpload = [&](const auto& simulationHandler) {
```
Comment (hernando) [inline plugins/CircuitViewer/tests/braynsTestData.cpp]: Did you forget how to declare a normal function? :)

File: plugins/CircuitViewer/tests/braynsTestData.cpp [hunk 2]
```diff
@@ -29,12 +29,35 @@
...
+    using namespace std::chrono_literals;
+
+    auto start = std::chrono::high_resolution_clock::now();
+    auto elapsed = 0.0;
+
+    while (elapsed < WAIT_MILLISECONDS && !simulationHandler->isReady())
```
Comment (hernando) [inline plugins/CircuitViewer/tests/braynsTestData.cpp]: Why not fixing it adding a waitReady function in the simulationHandler?

File: plugins/CircuitViewer/tests/braynsTestData.cpp (hunk 1 above)
Comment (karjonas) [reply plugins/CircuitViewer/tests/braynsTestData.cpp]: Yes :) this is now removed.

File: plugins/CircuitViewer/tests/braynsTestData.cpp (hunk 2 above)
Comment (karjonas) [reply plugins/CircuitViewer/tests/braynsTestData.cpp]: Done.

**Pull request merged**